doc/src/build/fullnode.md

```diff
@@ -0,0 +1,127 @@
+
+**Important**: For potential validators, running a Sui full node is an absolute prerequisite. We encourage auditors, bridges, state mirrors and other interested parties to join us. We offer no guarantees on performance or stability. We’re seeking feedback in the form of [issues filed in GitHub](https://github.com/MystenLabs/sui/issues/new/choose).
+
+## Features
+
+Sui full nodes exist to:
+
+* Track the state of the blockchain, independently and locally.
+* Serve read requests from clients.
+* Conduct local app testing against verified data.
+* [Reward full node runners for their service](../learn/tokenomics.md) with high-quality data.
+
+
+## State-Synchronization
+
+Today Sui full nodes sync with validators to be able to learn about newly committed transactions. 
+
+The normal life of a transaction requires a few round trips to 2f+1 validators to be able to form a TxCert, at which point a transaction is guaranteed to be committed and executed. 
+
+Today this synchronization process is performed by:
+
+1. Following 2f+1 validators and listening for newly committed transactions.
+2. Requesting the transaction from one validator.
+3. Locally executing the transaction and updating the local DB.
+
+Today this synchronization process is far from ideal as it requires listening to at a minimum 2f+1 validators to ensure that a full node has properly seen all new transactions. Overtime we will improve this process (e.g. with the introduction of a checkpoints, ability to synchronize with other full nodes, etc) in order to have better guarantees around a full node’s ability to be confident it has seen all recent transactions.
+
+## Architecture
+
+The Sui full node is essentially a read-only view of the network state. Unlike validator nodes, full nodes cannot sign transactions, although they can validate the integrity of the chain by re-executing transactions that were previously committed by a quorum of validators. 
+
+Today a full node is expected to maintain the full history of the chain, although in the future sufficiently old history may need to be pruned and offloaded to cheaper storage. 
+
+Conversely, a validator needs to store only the latest transactions on the *frontier* of the object graph (e.g., txes with >0 unspent output objects).
+
+## Full node setup
+
+Follow the instructions here to run your own Sui full node.
+
+### System requirements
+
+
+#### Hardware
+
+This is the minimum recommended hardware for running a Sui full node. In Amazon Web Services (AWS), this is known as a t2.large:
+
+* CPUs: 2
+* RAM: 8GB
+* Disk: 32GB SSD
+* Cost/hr: $0.09
```

Comment:
I was gonna say that it's probably better to provide a link to the t2.large instance type in AWS, rather than specifying a cost which is presumably in US$ here... but many people are not in the US, so having a link is better to let people translate to their own currency.